lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -86,11 +86,11 @@ def self.init_vmpooler()
         execute_rake_task("beaker_quickstart:gen_hosts[vmpooler]")
       end
 
-      # Print a message to the console and exit with 0
+      # Print a message to the console and exit with 1
       # @param [String] msg the message to print
```

Comment:
Along with the new docstring, is there a reason why this defaults to 1 instead of 0?